core/controllers/classroom.py

```diff
@@ -69,6 +70,8 @@ def get(self, classroom_name):
         ]
 
         self.values.update({
-            'topic_summary_dicts': topic_summary_dicts
+            'topic_summary_dicts': topic_summary_dicts,
+            'classroom_page_is_shown': (
```

Comment:
This doesn't seem like a good place for this value*

Reply:
Addressed.